src/astro/sql/table.py

```diff
@@ -52,6 +55,19 @@ def qualified_name(self):
     def __str__(self):
         return f"Table(table_name={self.table_name}, database={self.database}, schema={self.schema}, conn_id={self.conn_id}, warehouse={self.warehouse})"
 
+    def drop(self):
```

Comment:
Oh this is a nice touch! I like this idea of treating tables as object that have functionality